frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -171,6 +164,20 @@ action.GitMachete.SlideOutSelectedBranchAction.GitMacheteContextMenu.text=Slide
 action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected branch
 
 
+# Graph Table
+
+action.GitMachete.DiscoverAction.text=Discover Branch Layout
+action.GitMachete.DiscoverAction.description=Discovers the Branch Tree in Git Machete Plugin
+action.GitMachete.DiscoverAction.cant-discover-layout-error-title="Can't discover repository branch tree"
```

Comment:
Are the quotes needed? Won't they make their way all the way into the resulting displayed string?

Reply:
If they are needed, add a comment why needed here but not e.g. below in `DiscoverAction.write-file-error-title`

Reply:
Yeah, quotes are unneeded